components/Sidebar/ChatThreadList.tsx

```
@@ -0,0 +1,147 @@
+  };
+
+  return (
+    <div className="flex flex-col h-full">
+      <ScrollArea className="flex-1 overflow-y-auto">
+        <div className={collapsed ? "p-1" : "p-2 pt-0"}>
+          {loading ? (
+            <div className="text-center py-5">
+              {!collapsed && (
+                <>
+                  <div className="animate-spin h-5 w-5 border-2 border-primary rounded-full border-r-transparent mx-auto mb-2" />
+                </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)